saleor/graphql/account/mutations/account/account_update.py

```diff
@@ -91,3 +94,24 @@ def perform_mutation(cls, root, info: ResolveInfo, /, **data):
         user = cast(models.User, user)
         data["id"] = graphene.Node.to_global_id("User", user.id)
         return super().perform_mutation(root, info, **data)
+
+    @classmethod
+    @traced_atomic_transaction()
+    def save(cls, info: ResolveInfo, instance: models.User, cleaned_input):
+        manager = get_plugin_manager_promise(info.context).get()
+
+        default_addresses = cls.save_and_apply_addresses_from_input(
+            cleaned_input=cleaned_input, user_instance=instance
+        )
+
+        super().save(info, instance, cleaned_input)
```

Comment:
Do we need to call the `save` of super class? It's probably saving all object (all fields instead of that that have been changed). 
But maybe is a topic for a separate PR

Reply:
Also the save is called twice (line 107 and 115), and probably can be called once